### DISCUSSION TITLE
Drag and Drop issues on Linux

### DISCUSSION THREAD

**Discussion by theodditie** [burkybang/SuiteAdvanced-Feedback#155] [Q&A] [answered]
I've been using SuiteAdvanced for almost a year now and love it. One of the features I use frequently is drag and drop for the file cabinet to upload product images from my file browser on my computer, and usually don't have any issues. I learned early on that it works with files, but not folders.

I have used this on Windows 11, Ubuntu, and Garuda Linux, but recently I switched to using Fedora instead. Today I tried to drag and drop from Dolphin file browser, same one I used in Garuda, but every time I immediately get an error.

Has anyone else had any issues with drag and drop?

**Comment by burkybang** [comment]
What error are you getting?

**Reply by theodditie** [reply]
It just says Upload Error and the file name, no other details.

**Reply by burkybang** [reply]
Okay, I'm installing Fedora on a virtual machine to see if I can reproduce the issue.

**Reply by burkybang** [reply]
@theodditie What browser are you using on Fedora?

**Reply by theodditie** [reply]
Chrome Version 126.0.6478.61

**Reply by theodditie** [reply]
Running the Gnome environment, with Dolphin File Browser. Using the default Files browser, I don't get the error, I get the success message but no file actually uploaded.

I prefer the Dolphin browser because, much like my web browser, I usually have multiple tabs open on it, but thought I would give the default browser a chance.

**Reply by burkybang** [reply]
Sadly, I wasn't able to reproduce the issue. I wonder what's different between our setups that's causing an issue for you.

![image](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/927886/aaf140b7-23cf-4d28-b06c-9f76cd073892)

**Reply by theodditie** [reply]
That's usually my luck. I've even tried to use KDE Plasma since that was the same as what Garuda uses, but still no luck. If I knew how to find out what the error actually is, that could possibly help figure out what's different on my end.

**Reply by burkybang** [reply]
Send me an email (support@suiteadvanced.com), and we can set up a call to debug it together.

**Reply by theodditie** [reply, marked as answer]
As I was about to send you an email, I remembered something about Fedora that I haven't messed with a lot: Flatpaks. I had installed Chrome through the store, which does it as a Flatpak and prevented access to the files I was trying to drag and drop.

After removing the Flatpak Chrome and installing through the .rpm from Chrome's website, drag and drop is working again!

**Reply by burkybang** [reply]
Oh fantastic! So glad you figured it out
